policyengine_us/parameters/gov/states/ar/tax/income/deductions/itemized/medical_expense_rate.yaml

```diff
@@ -0,0 +1,21 @@
+description: Arkansas limits the medical and dental expenses to this rate of adjusted gross income.
+values:
+  2021-01-01: 0.1
```

Comment:
@MaxGhenis should we use this created parameter for the 2013 medical floor or the federal one? (It has since been raised which does not apply to AR) - if so how could we access it?